src/main/java/dk/kb/datahandler/oai/OaiHarvestClient.java

```diff
@@ -231,11 +233,21 @@ protected PasswordAuthentication getPasswordAuthentication() {
                 }).build();
 
 
+        /* The Preservica 7 OAI-PMH endpoint is acting up. Per their documentation they say that they follow the standard
+            and makes Basic authorixation possible. However, the response does not contain a WWW-Authenticate header, when queried.
+
+            If using their acces endpoint to get an access-token then the service works. I got an access token by calling the following:
+            curl -X 'POST'   'DEVELSERVER'   -H 'accept: application/json'   -H 'Content-Type: application/x-www-form-urlencoded'   -d 'username=USERNAME&password=PASSWORD&cookie=false&includeUserDetails=false'
+            Changing DEVELSERVER, USERNAME and PASSWORD
 
+            I've written the integration test dk.kb.datahandler.oai.OaiHarvestClientIntegrationTest.testPreservicaSevenAuth
+            which makes it seem like this might be a problem in our end as the test parses.
+         */
```

Comment:
Is this still relevant after our pair-review test?

Reply:
Nope! Thanks!